procedures/automation-android-setup.md

```diff
@@ -0,0 +1,200 @@
+13. Once step 11's PR is approved and the build is working in step 12, merge the `build-puppet` PR
+14. Verify with app's team how `versionCode` should be set up. Perhaps by date like [`fenix`](https://github.com/mozilla-mobile/fenix/blob/master/automation/gradle/versionCode.gradle)?
+    * Note that if there's multiple build types, they need different version codes. In the case of [`fenix`](https://github.com/mozilla-mobile/fenix/blob/master/app/build.gradle#L50-L52), `x86` builds have the version code incremented by 1.
+15. When the Google Play product is being set up, an officially-signed build with a version code of 1 needs to be built. So, the main automation PR for the product will need to be stunted: it needs to produce APKs with a version code of 1, and it should have pushing to Google Play disabled (so we don't accidentally push a build before our official version-code-1 build is set up).
+    1. Change the version code to be set to 1. If the product uses the same version-code-by-date schema as `fenix`, then edit [`versionCode.gradle`](https://github.com/mozilla-mobile/fenix/pull/156/files#diff-63606bb315fadc051f73a54767849985R41)
+    2. [Disable the creation of the task that pushes to Google Play](https://github.com/mozilla-mobile/fenix/pull/156/files#diff-73e51d972c105de5122ec559909980daR123)
+    3. Create the PR
+    4. Once approved, merge the PR
+16. Verify the apk artifact(s) of the signing task
+    1. Trigger the non-staging hook
+    2. Once the build finishes, download the apks from the signing task
+    3. Using the prod certificate from step 10.4.1., create a temporary keystore: `keytool -import -noprompt -keystore tmp_keystore -storepass 12345678 -file $product_release.pem -alias $product-rel`
+    4. For each apk, verify that it matches the certificate: `jarsigner -verify $apk -verbose -strict -keystore tmp_keystore`. Check that
+        * The "Digest algorithm" matches step 1
+        * The "Signature algorithm" matches step 1
+        * There are no warnings that there are entries "whose certificate chain invalid", "that are not signed by alias in this keystore" or "whose signer certificate is self-signed"
+    * Do the same thing for the dep signing task and certificate and check that the `jarsigner` command shows that the "Signed by" `CN` is "Throwaway Key"
+17. Request both the creation of a Google Play product and for the credentials to publish to it. Consult with the product team to [fill out the requirements for adding an app to Google Play](https://wiki.mozilla.org/Release_Management/Adding_a_new_app_on_Google_play). This request should be a bug for "Release Engineering > Release Automation: Pushapk", and should be a combination of [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1508294) and [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1512173)
+    * As part of the bug, note that you'll directly send an APK to Sylvestre Ledru and Julien Cristau via Slack
```

Comment:
Nit: We may want to replace "Sylvestre and Julien" by "the point of contact in release management". We don't know for sure whether Sylvestre and Julien will remain the point of contacts forever :slightly_smiling_face: